tests/ImplementationLoader.php

```diff
@@ -54,6 +53,11 @@ protected function __construct(Connection $connection, $fixturePath)
                     'Query\\XPath', // Query language 'xpath' not implemented.
                     'Query\\Sql1', // Query language 'sql' is legacy and only makes sense with jackrabbit
                     'Writing\\CloneMethodsTest', // TODO: Support for workspace->clone, node->update, node->getCorrespondingNodePath
+
+                    // TODO fully implement versioning
```

Comment:
We are, but I and @dbu agreed on trying to merge always parts, and skip the tests which are not succeeding yet. This way it's easier for him to review.